example/README.md

```diff
@@ -3,6 +3,4 @@
 | Platform | Example interface                                                      | Sample tweet                                                           |
 |:--------:|:--------------------------------------------------------------------|:------------------------------------------------------------------------|
 | Android  | ![img](https://raw.githubusercontent.com/schibsted/tweet_ui/master/example/screenshots/android_main.png) | ![img](https://raw.githubusercontent.com/schibsted/tweet_ui/master/example/screenshots/android_example.png) |
-|   iOS    | ![img](https://raw.githubusercontent.com/schibsted/tweet_ui/master/example/screenshots/ios_main.png)    | ![img](https://raw.githubusercontent.com/schibsted/tweet_ui/master/example/screenshots/ios_example.png)    |
-
-> Video/GIF Tweets don't work on iOS simulators ([video_player ios instructions](https://pub.dev/packages/video_player#ios))
```

Comment:
Is this sentence not true anymore:
> Video/GIF Tweets don't work on iOS simulators 

If it's still valid, we should leave it

Reply:
Reverted it